apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.ts

```diff
@@ -225,8 +225,8 @@ export class DaybookAdvanceSearchModelComponent implements OnInit, OnChanges, On
                 dataToSend.dateOnCheque = moment(dataToSend.dateOnCheque, GIDDH_DATE_FORMAT).format(GIDDH_DATE_FORMAT);
             }
         }
-        let fromDate = this.fromDate;
-        let toDate = this.toDate;
+        let fromDate = this.fromDate ? this.fromDate : this.selectedDateRange?.startDate ? this.selectedDateRange.startDate.format(GIDDH_DATE_FORMAT) : "";
```

Comment:
Consider this for both `fromDate` and `toDate`
```suggestion
        let fromDate = this.fromDate ?? this.selectedDateRange?.startDate ?? this.selectedDateRange.startDate.format(GIDDH_DATE_FORMAT);
```